Web/stockdogweb/src/components/Graph/Graph.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { Line } from 'react-chartjs-2';
+import "./Graph.css";
+
+import loading from "../../img/loading.svg";
+
+class Graph extends Component {
+   loadingAnimation = (
+      <div className="Graph-loading-animation-wrapper">
+         <div className="Graph-loading-animation">
+            <img src={loading} alt="Loading" />
+         </div>
+      </div>
+   )
+   
+   data = {
+      labels: this.props.labels,
+      datasets: [
+         {
+            fill: false,
+            lineTension: 0,
+            borderColor: 'rgb(247, 248, 249)',
```

Comment:
Would you want to set this color as a variable since you're using it multiple times?